### DISCUSSION TITLE
✅ Creating an OSPO Local Meetup - Spain (Spanish Speaking)

### DISCUSSION THREAD

**Discussion by miguelsalva** [todogroup/ospology#227] [Community Proposals]
Hi all,

I feel the creation of a OSPO Local Meetup in Spain (probably Madrid) will benefit not only a lot of Spanish companies that have not fully engaged in open source with the right organization, but also to a lot of companies with established OSPOs so they can learn from each other and share their best practices. I know there are some of them already involved in this community.

Maybe we can start with the creation of the slack channel **#chapter-spain** and see how things go from there. If we some movement and/or initiatives, we can organize a face-to-face meeting.

OSPO local Chapter organizer: @anajsana , @miguelsalva

**Comment by anajsana** [comment]
Thanks for submitting the requests @miguelsalva . As we discussed in previous conversations, I'm happy to be an OSPO Local Organizer of these meetups. I'll share this proposal to the TODO SC for review and approval 👍 

Regarding the name for the slack channel, I think we should decide if the PRIMARY purpose is to build conversations around specific region problems within the same timezone (in that case, using the word *Spain* would make more sense, or *Ibero* if we want to include Portuguese organizations) or to build content around a common language, that can also open the door for other Hispanic Countries in the Americas (and here maybe *Hispanic* would make more sense (?))

**Reply by miguelsalva** [reply]
Thanks a lot for the help @anajsana :) 

I would say let's start slow and gain momentum if required: my proposal is using #chapter-spain for people within the same timezone and also using Spanish as the common language. Of course, everyone is welcome and if the situation changes and the chapter grows, we will be able to accommodate

**Comment by juliancoccia** [comment]
That sounds good. Please count me in! 

Julian Coccia
CTO

Scan Open Source Solutions SL
Paseo de la Castellana 81, Planta 15
28046, Madrid
Spain
+34 683 383 936


￼
scanoss.com <https://scanoss.com/>

**Comment by anajsana** [comment]
Good news @miguelsalva! The review process has been completed by SC and the `OSPO Local Meetup - Spain (Spanish Speaking)` proposal is approved! Below are essential details for you to know:

## Registration Platform Acess (not ready yet)

- [ ] Create an OSPO Local Meetup - Spain chapter via LF Online Community Platform 
> IMPORTANT: This will be done in the future, once the slack community starts to grow

## Real-time Communication Slack

- [ ] Create a new slack channel in the TODO slack called #chapter-spain

## Meetup Summaries & Documentation

OSPO Local organizers should write a summary of what is being done / accomplished during these meetups on a quarterly basis via OSPOlogy GH Discussions. An example can be found at: https://github.com/todogroup/ospology/discussions/212
